sql/moz-fx-data-shared-prod/static/README.md

```diff
@@ -16,3 +16,6 @@ ETL and analysis:
 ./script/publish_static --project-id mozdata
 ./script/publish_static --project-id moz-fx-data-shared-prod
 ```
+
+Note the `country_names` `data.csv` is generated/updated using an SQL generator:
+`bqetl generate country_code_lookup`.
```

Comment:
```suggestion
```

I think we should move this into the bash snippet above to make it less likely to get skipped.